plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/totals/footer-item/index.tsx

```diff
@@ -140,30 +140,33 @@ const TotalsFooterItem = ( {
 			value={ value }
 			description={
 				<>
-					{ SHOW_TAXES &&
-						parsedTaxValue !== 0 &&
-						( isLoading ? (
-							<Skeleton
-								width="45px"
-								height="1.55em"
-								ariaMessage={ __(
-									'Loading price… ',
-									'woocommerce'
-								) }
-							/>
-						) : (
-							<p className="wc-block-components-totals-footer-item-tax">
-								{ createInterpolateElement( description, {
+					{ SHOW_TAXES && parsedTaxValue !== 0 && (
+						<p className="wc-block-components-totals-footer-item-tax">
+							{ isLoading ? (
+								<>
+									<span>Including </span>
```

Comment:
Should we translate the whole text? We risk a situation in which the real message will be in a different language vs the loading state hardcoded to English 🤔